low-level/json_msgs/messages/sensors/realstor_disk_group_data.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) 2020-2021 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+import json
+from json_msgs.messages.sensors.base_sensors_msg import BaseSensorMsg
+
+
+class RealStorDiskGroupDataMsg(BaseSensorMsg):
+    '''The JSON message transmitted by the Service Watchdogs.'''
+
+    SENSOR_RESPONSE_TYPE = "enclosure_disk_group_alert"
+    MESSAGE_VERSION = "1.0.0"
+
+    def __init__(self, host_name,
+                 alert_type,
+                 alert_id,
+                 severity,
+                 info,
+                 specific_info,
+                 username="SSPL-LL",
+                 signature="N/A",
+                 time="N/A",
+                 expires=-1):
+        '''Initialization of RealStorDiskGroupDataMsg.'''
+
+        super(RealStorDiskGroupDataMsg, self).__init__()
+
+        # Header attributes
+        self._alert_type = alert_type
+        self._host_name = host_name
+        self._alert_id = alert_id
+        self._severity = severity
+        self._username = username
+        self._signature = signature
+        self._time = time
+        self._expires = expires
+        self._fru_info = info
```

Comment:
Do we need these assignments to self variables??
Probably it is good to use the function args directly as well right?

Reply:
same comment https://github.com/Seagate/cortx-monitor/pull/519/files/374f2d7375f4a7fa82e4c883b01ad196ca0bc709#r699552651